cdflow_commands/config.py

```diff
@@ -54,17 +59,25 @@ def load_service_metadata():
 
 
 def load_global_config(account_prefix, aws_region):
+    ecs_cluster = 'default'
+    ecs_cluster_key = 'ecs_cluster.{}.name'.format(ecs_cluster)
```

Comment:
This currently pulls the cluster name out of the platform config. Instead we are going to remove the indirection so the value of `ecs_cluster` will be the actual name of the ecs cluster.